README.md

```diff
@@ -14,6 +14,7 @@ Demonstrator tracking chain for accelerators.
 |                    | Track param estimation | ✅  | ✅   | ✅   | ✅     | ⚪     | ⚪      |
 | **Track finding**  | Combinatorial KF       | ✅  | ✅   | 🟡   | 🟡     | ⚪     | ⚪      |
 | **Track fitting**  | KF                     | ✅  | ✅   | ✅   | ⚪     | ⚪     | ⚪      |
+| **Ambiguity resolver**  | Greedy resolver   | ✅  | ⚪   |  ⚪  | ⚪     | ⚪     | ⚪      |
```

Comment:
In keeping with the phrasing of the other rows, let's call it "Ambiguity resolution".

But yes, I'm very much on board!